darts/logging.py

```diff
@@ -57,6 +57,7 @@ def raise_deprecation_warning(
 
 def raise_if_not(
     condition: bool,
+    *,
```

Comment:
Why is this needed now?

Reply:
It is to force using parameter name, so that the caller has to use `raise_if_not(..., msg="msg")` instead of `raise_if_not(..., "msg")`. If you think this style is bad, I can remove it.

Reply:
OK I see. I think it'd be slightly better to remove it so we keep things a bit simpler. There's no need to enforce the keyword arguments too strongly IMO.